.solhint.json

```diff
@@ -2,6 +2,7 @@
   "extends": "solhint:default",
   "plugins": ["prettier"],
   "rules": {
-    "prettier/prettier": "error"
+    "prettier/prettier": "error",
+    "max-line-length": ["warn",120]
```

Comment:
I think 140 is better.

Reply:
Updated to 140 in the latest commit aa878fd